package.json

```diff
@@ -11,7 +11,7 @@
     "start": "./bin/http-mockserver.js",
     "debug": "node --inspect --debug-brk src/server/index.js",
     "lint": "eslint ./src",
-    "test": "jest"
+    "test": "NODE_ENV=test jest"
```

Comment:
i thought jest automatically sets this (the note in http://facebook.github.io/jest/docs/getting-started.html#additional-configuration)